### ISSUE TITLE
508-defect-2 [AXE-CORE]: Heading levels SHOULD increase by one - sidebar Message us, Call us, etc

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9981]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Page(s) affected: https://www.va.gov/decision-reviews/

Headings on a web page **should** progress, increasing by one, in sections of content. The headings in the sidebar are `h4`, but the previous heading is an `h2`. 

> Note: The sidebars weren't rendered when this work was audited in the VA environment during the staging review.

### Why it Matters
The underlying purpose of headers is to convey the structure of the page. For sighted users, the same purpose is achieved using different sizes of text. Text size, however, is not helpful for users of screen readers, because a screen reader identifies a header only if it is properly marked-up. When heading elements are applied correctly, the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements applied in hierarchical order. All of these qualities make headings valuable tools for screen reader users. Similar to the way sighted users can glance at a page and get a sense of its contents, screen reader users can navigate through headings. Well written and properly ordered headings can save screen reader time and frustration.

In addition to making the page more accessible, headings have other benefits since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging & production

## Steps to Recreate

1. Enter https://www.va.gov/decision-reviews/ in browser
2. Run axe scan
3. Verify the issue shown in the screenshot is logged

## Possible Fixes (optional)

For the sidebar, use `h2` with an `h4` utility class, or use a `dl` to structure the sidebar content.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order)
* [MDN dialog element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog)
* [MDN ARIA: alert role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role)
* [MDN Description List element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl)

## Screenshots or Trace Logs

<img width="1412" alt="Screen Shot 2020-06-09 at 12 46 54 PM" src="https://user-images.githubusercontent.com/57469/84176403-539a9780-aa4f-11ea-8798-b2dac55c04a8.png">
